src/com/mercado/model/product/Ingredient.java

```diff
@@ -0,0 +1,31 @@
+package com.mercado.model.product;
+
+public class Ingredient {
+    private int id;
```

Comment:
Ok , but why?, explain your review

Reply:
Java included a lot of features on its core that you are not taking advantage of, just because of primitive values. 

1. You are preventing your code from using latest java features in the first try of your code.
2. Java implementation knows that there could be performance issues with their implementation, hence they have imnprove the compiler to box and unbox primitive values as well was parsing them.
3. Taking advantage of other java features example Collections, Streaming, Futures, Optionals and Functions might make you create extra verbose code.

In summary, use primitive only if the implementation of the java core is affecting you performance

Reply:
ok, thanks